pom.xml

```diff
@@ -888,7 +888,18 @@
                   </ignoreClasses>
                   <findAllDuplicates>true</findAllDuplicates>
                   <ignoreWhenIdentical>true</ignoreWhenIdentical>
+                  <dependencies>
+                    <dependency>
+                      <groupId>net.snowflake</groupId>
+                      <artifactId>snowflake-jdbc</artifactId>
+                      <ignoreClasses>
```

Comment:
do you know if jdbc is pulling in protobuf too? can we ignore that?
https://github.com/snowflakedb/snowflake-ingest-java/issues/876

Reply:
This ignore is just for the `banDuplicateClasses` enforcer check, it does not exclude any classes from the JAR. Until we move to `snowflake-jdbc-thin`, we have no way of excluding classes from the JDBC driver.